components/hab-spider/src/main.rs

```diff
@@ -22,38 +22,53 @@ extern crate walkdir;
 extern crate habitat_core as hab_core;
 extern crate builder_core as bldr_core;
 extern crate habitat_builder_protocol as protocol;
+extern crate habitat_builder_db as db;
+extern crate habitat_net as hab_net;
 extern crate clap;
+extern crate postgres;
+extern crate protobuf;
+extern crate r2d2;
+
+pub mod data_store;
+pub mod error;
 
 use std::io::{self, Write};
 use clap::{Arg, App};
 use time::PreciseTime;
 use bldr_core::package_graph::PackageGraph;
-use bldr_core::file_walker::FileWalker;
+//use bldr_core::file_walker::FileWalker;
```

Comment:
I left it in as we may add the ability to 'migrate' on-disk packages from hab-spider as well. I'll remove it for now though.